modules/wr_tripower9000/tripower.py

```diff
@@ -58,7 +58,7 @@ def DebugLog(message):
     counter = 0
 
     # Länge von sys.argv = Dateiname + Webbox + IP-Adressen
-    for i in len(sys.argv)-2:
+    for i in range(len(sys.argv)-2):
```

Comment:
Es gäbe auch `range(2, len(sys.argv))`

Noch sinnvoller wäre: `for ipaddress in sys.argv[2:]:`